dataplane/fragmentation.h

```diff
@@ -37,30 +38,38 @@ using value_t = std::tuple<std::map<uint32_t, ///< range_from
                            uint16_t, ///< first packet time
                            uint16_t>; ///< last packet time
 
-}
-
-class fragmentation_t
+class Fragmentation
 {
 public:
-	fragmentation_t(cControlPlane* controlPlane, cDataPlane* dataPlane);
-	~fragmentation_t();
+	using OnCollected = std::function<void(rte_mbuf*, const common::globalBase::tFlow&)>;
+	Fragmentation(OnCollected callback);
+	Fragmentation(OnCollected callback, const FragmentationConfig& cfg);
+	Fragmentation(Fragmentation&& other);
+	~Fragmentation();
+
+	Fragmentation& operator=(Fragmentation&& other);
 
 public:
-	common::fragmentation::stats_t getStats();
+	common::fragmentation::stats_t getStats() const;
+	OnCollected& Callback() { return callback_; }
+	void Configure(const FragmentationConfig& cfg) { config_ = cfg; }
 
 	void insert(rte_mbuf* mbuf);
 	void handle();
 
 protected:
-	bool isTimeout(const fragmentation::value_t& value);
```

Comment:
Unrelated, but we're making this class better in this commit anyway, so I guess it's fine:
Let's change `protected` to `private` here. We're not using this class in inheritance in the project